PyInstaller/depend/dylib.py

```diff
@@ -167,10 +177,38 @@ def search(self, libname):
     from PyInstaller.lib.macholib import util
 
     class MacExcludeList(object):
+        def __init__(self, global_exclude_list):
+            # Wraps the global 'exclude_list' before it is overriden
+            # by this class.
+            self._exclude_list = global_exclude_list
+
         def search(self, libname):
-            return util.in_system_path(libname)
+            # First try global exclude list. If it matches then
+            # return it's result otherwise continue with other check.
+            result = self._exclude_list.search(libname)
+            if result:
+                return result
+            else:
+                return util.in_system_path(libname)
+
+    exclude_list = MacExcludeList(exclude_list)
+
+elif is_win:
+    class WinExcludeList(object):
+        def __init__(self, global_exclude_list):
+            self._exclude_list = global_exclude_list
+            self._windows_dir = winutils.get_windows_dir()
 
-    exclude_list = MacExcludeList()
+        def search(self, libname):
+            result = self._exclude_list.search(libname)
+            if result:
+                return result
+            else:
+                # Exclude everything from the Windows directory by default.
+                fn = os.path.realpath(libname)
+                return fn.startswith(self._windows_dir)
```

Comment:
Fine. But where is the code? :-) Did you miss pushing your changes to github? :-)